src/math/linalg/dir/dir2.rs

```diff
@@ -51,6 +54,35 @@ impl From<Unit<Vector2<Real>>> for Dir2 {
     }
 }
 
+impl Neg for Dir2 {
```

Comment:
At the point where you have to implement all these things for the wrapped `Dir2` class... it makes me wonder if it's too much hassle to have a wrapper instead of an alias?

Do you add/restrict functionality to `Dir2` then it's worth it in the long run? If not then it's a lot of wrapping code

Reply:
I agree, some of the bugs I was trying to hunt down were because I was trying to fix implementation issues in the wrapper. We can discuss this further if useful.